js/payment-methods/lightning.js

```diff
@@ -0,0 +1,117 @@
+var app = app || {};
+
+app.paymentMethods = app.paymentMethods || {};
+
+app.paymentMethods.lightning = (function() {
+
+    'use strict';
+
+    return app.abstracts.PaymentMethod.extend({
+	// The name of the cryptocurrency shown in the UI:
+	label: 'Lightning',
+
+	// The exchange symbol:
+	code: 'BTC',
+
+	// Used internally to reference itself:
+	ref: 'lightning',
+
+	lang: {
+	    'en' : {
+		'settings.memo.label': 'Invoice Memo',
+	    	'invalid-payment-request': 'Invalid payment request',
+	    }
+	},
+
+	settings: [
+	    {
+	    	name: 'memo',
+	    	label: function() {
+	    		return app.i18n.t('lightning.settings.memo.label');
+	    	},
+	    	type: 'text',
+	    	required: false,
+	    } 
+	],
+
+
+	generatePaymentRequest: function(amount, cb) {
+
+	    var satoshi = parseInt(amount*1e8); // convert to satoshi
```

Comment:
parseInt should have radix for base10 as well.
`var satoshi = parseInt(amount * 1e8, 10);`